yb-voyager/src/dbzm/schemaRegistry.go

```diff
@@ -56,36 +61,36 @@ func (ts *TableSchema) getColumnType(columnName string) (string, error) {
 //===========================================================
 
 type SchemaRegistry struct {
-	exportDir         string
+	exportDir             string
 	exporterRole      string
 	tableNameToSchema map[string]*TableSchema
 }
 
 func NewSchemaRegistry(exportDir string, exporterRole string) *SchemaRegistry {
 	return &SchemaRegistry{
-		exportDir:         exportDir,
+		exportDir:             exportDir,
 		exporterRole:      exporterRole,
 		tableNameToSchema: make(map[string]*TableSchema),
 	}
 }
 
-func (sreg *SchemaRegistry) GetColumnTypes(tableName string, columnNames []string) ([]string, error) {
+func (sreg *SchemaRegistry) GetColumnTypes(targetDBType, tableName string, columnNames []string) ([]string, error) {
```

Comment:
consider setting targetDBType at the schemaRegistry level, so you won't have to change these   APIs.